ecms_base/ecms_base.profile

```diff
@@ -23,4 +23,11 @@ function ecms_base_update_9001(array &$sandbox): void {
   $active_storage->write('editor.editor.basic_html', $install_source->read('editor.editor.basic_html'));
   $active_storage->write('filter.format.basic_html', $install_source->read('filter.format.basic_html'));
 
+  // Make sure the scheduler module is installed.
+  \Drupal::service('module_installer')->install(['scheduler']);
+
+  // Call the workflow service to update configuration.
+  \Drupal::service('ecms_workflow.bundle_create')
+    ->assignWorkflowToActiveTypes();
+
```

Comment:
I believe update 9001 was run with tag 0.1.2. We should probably add these to a new update function. We should probably start numbering these based on the next coming tag too. So, 9013 would be next since 0.1.3 is the next coming tag.